src/ensembl/src/content/app/browser/drawer/drawer-views/transcript-summary/TranscriptSummary.scss

```diff
@@ -65,6 +65,10 @@
   font-weight: $light;
 }
 
+.leftSpacing {
```

Comment:
Unless we are repeating this class in multiple places in this view (which I don't think we are), better give it a more concrete name (`.geneSymbol`), add it to the `span` surrounding the gene symbol, and use a right margin instead of the left. So that when a gene does not have a symbol, the whole gene symbol span, with this class, won't even appear in the DOM.

Reply:
Done @azangru . Let me know if its all good now. Thanks